Magic Afternoons/Mob-Dojo.md

```diff
@@ -0,0 +1,82 @@
+# Mob Dojo Activity
+
+Mob programming is best defined as a software development approach where a team works on the same thing, at the same time, in the same space, and on the same computer. It is very similar to pairing, and involves more people.
+
+Mob programming naturally favors the Lean concept of flow efficiency rather than resource efficiency. So if you value getting things done and delivered to your customers, then yes, mobbing is very productive.
+
+This afternoon the objective is to complete a coding exercise using some very specific mob programming techniques.
+
+We will form groups with a minimum of 4 members and a maximum of 6 per mob and follow these rules:
+
+## Rules
+
+The team will start with 3 roles:
+
+* **Driver** - acts as a "smart keyboard"; she should not type whatever she wants, but interprets the direction of the Navigator(s) into code. The `Navigator(s)` thoughts must travel through the driver.
+
+* **Navigator** - you will communicate your thoughts to the `Driver`.  Your thoughts have to go through their hands to become code.
+
+* **Observer** - you will pay close attention and contribute your ideas and suggestions to other members of the mob based on the rules in each round.
+
+### Round 1 - Practice Patience
+
+Rules for this round are: Only one Navigator can talk. Everyone else observes. The Driver may not type any code without explicit direction from the single Navigator. Navigator you will verbalize your thoughts so your ideas flow through the Driver and become code in the computer. Every 5 minutes everyone will rotate until each person has been a Driver, Navigator, and Observer.  During this round Observers should not speak. Instead they should pay close attention and follow what is happening so they are prepared for the next role change.
```

Comment:
Missing a comma: "Navigator**,** you will verbalize your thoughts so your ideas flow through the Driver and become code in the computer"